nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/http1/ClientResponseImpl.java

```diff
@@ -122,6 +130,19 @@ public void close() {
         }
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public <T extends Source<?>> void source(GenericType<T> sourceType, T source) {
+        List<SourceHandler> providers = SOURCE_HANDLER_LOADER.asList();
```

Comment:
the `asList` should be called on the constant already, there is no need to delay it to runtime (for each request to this method)